app/partials/app.cjsx

```diff
@@ -3,15 +3,81 @@ auth = require 'panoptes-client/lib/auth'
 IOStatus = require './io-status'
 MainHeader = require './main-header'
 MainFooter = require './main-footer'
+GeordiClient = require 'zooniverse-geordi-client'
 {generateSessionID} = require '../lib/session'
 
+class GeordiLogger
+  constructor: (@state, @geordi) ->
+
+  keys: {}
+
+  geordi: null
+
+  instance: =>
+    @geordi = @geordi || @makeGeordi @keys?.projectToken
+
+  makeGeordi: (projectSlug) ->
+    new GeordiClient
+      server: @state?.env
+      projectToken: projectSlug || ''
```

Comment:
PS @eatyourgreens That sounds sensible about `panoptes-client`. Where is that, I don't immediately see it? 
We shouldn't tie Geordi client to Panoptes, as it is used with Ouroboros too - and could be used elsewhere too - but in terms of its specific integration to PFE that should certainly be consistent with other clients used in PFE.